vpm.go

```diff
@@ -29,7 +29,12 @@ func (v *vpm) getPluginsMatchingAny(pluginNames ...string) (plugins []string) {
 }
 
 func (v *vpm) addPlugins(pluginNames ...string) (err error) {
-	for _, pluginKey := range v.getPluginsMatchingAny(pluginNames...) {
+	plugins := v.getPluginsMatchingAny(pluginNames...)
+	if len(plugins) == 0 {
+		return fmt.Errorf("No plugin found matching %v", pluginNames)
+	}
+
+	for _, pluginKey := range plugins {
```

Comment:
Throw an error if no plugins match, this prevents a typo causing a failure to update and giving similar output